src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import BREAKPOINT from "../../../variables/breakpoint";
+import FONTFAMILY from "../../../variables/font_family";
+const Input = ({ defaultValue = "", onEditComplete }) => {
+  const ref = useRef();
+
+  useEffect(() => {
+    ref.current.value = defaultValue;
+    ref.current.onblur = (e) => onEditComplete(e.current.value);
+    ref.current.focus();
+    ref.current.onkeypress = (e) => {
+      if (e.key == "Enter") {
```

Comment:
```suggestion
      if (e.key === "Enter") {
```
可能な限り厳密等価演算子を使っていきましょう。
参考：https://zenn.dev/bokutano26/articles/442f11f4ec69bf